jerry-debugger/jerry_client_main.py

```diff
@@ -734,10 +736,10 @@ def process_messages(self):
                 return DebuggerAction(DebuggerAction.TEXT, result)
 
             elif buffer_type == JERRY_DEBUGGER_EXCEPTION_STR:
-                self.exception_string += data[1:]
+                self.exception_string += data[1:].decode('latin1')
 
             elif buffer_type == JERRY_DEBUGGER_EXCEPTION_STR_END:
-                self.exception_string += data[1:]
+                self.exception_string += data[1:].decode('latin1')
```

Comment:
Is Latin-1 encoding OK for exception strings? Cannot they contain arbitrary Unicode characters?

Btw, this could be simplified to `elif buffer_type in [JERRY_DEBUGGER_EXCEPTION_STR, JERRY_DEBUGGER_EXCEPTION_STR_END]:`, just like elsewhere in the code.

Reply:
Yes utf-8 should be used.

same reason as here: https://github.com/jerryscript-project/jerryscript/pull/5095#discussion_r1330545099